src/Raven.Server/Smuggler/Documents/Handlers/SmugglerHandler.cs

```diff
@@ -63,29 +63,8 @@ public async Task PostExport()
         [RavenAction("/databases/*/admin/smuggler/import", "GET", AuthorizationStatus.DatabaseAdmin, DisableOnCpuCreditsExhaustion = true)]
         public async Task GetImport()
         {
-            if (HttpContext.Request.Query.ContainsKey("file") == false &&
-                HttpContext.Request.Query.ContainsKey("url") == false)
-            {
-                throw new ArgumentException("'file' or 'url' are mandatory when using GET /smuggler/import");
-            }
-
-            using (ContextPool.AllocateOperationContext(out DocumentsOperationContext context))
-            {
-                var options = DatabaseSmugglerOptionsServerSide.Create(HttpContext);
-
-                await using (var stream = new GZipStream(new BufferedStream(await GetImportStream(), 128 * Voron.Global.Constants.Size.Kilobyte), CompressionMode.Decompress))
-                using (var token = CreateOperationToken())
-                using (var source = new StreamSource(stream, context, Database.Name))
-                {
-                    var destination = Database.Smuggler.CreateDestination();
-
-                    var smuggler = Database.Smuggler.Create(source, destination, context, options, token: token.Token);
-
-                    var result = await smuggler.ExecuteAsync();
-
-                    await SmugglerHandlerProcessorForImport.WriteSmugglerResultAsync(context, result, ResponseBodyStream());
```

Comment:
you missed this? I do not see that code in the abstract processor?

Reply:
Yes, fixed